doc/1chipML_manual.tex

```diff
@@ -20,7 +20,7 @@
 
 \begin{figure}[h]
 \centering
-\includegraphics[width=0.3\textwidth]{1chipML_logo.eps}
+\includegraphics[width=0.3\textwidth]{resources/1chipML_color.eps}
```

Comment:
Is there a reason why we added a folder for the resources? I don't think it's a bad idea but since the image is not on the repo I don't see the point of it.